cardano-node/src/Cardano/Node/Configuration/LedgerDB.hs

```diff
@@ -20,17 +24,24 @@ import           Ouroboros.Consensus.Storage.LedgerDB.HD.LMDB (LMDBLimits (..))
 --
 -- See 'Ouroboros.Consnesus.Storage.LedgerDB.OnDisk.BackingStoreSelector'.
 data BackingStoreSelectorFlag =
-    LMDB (Maybe Int) -- ^ A map size can be specified, this is the maximum disk
-                     -- space the LMDB database can fill. If not provided, the
-                     -- default of 12Gi will be used.
+    LMDB (Maybe Gigabyte) -- ^ A map size can be specified, this is the maximum
+                          -- disk space the LMDB database can fill. If not
+                          -- provided, the default of 16GB will be used.
   | InMemory
   deriving (Eq, Show)
 
+newtype Gigabyte = Gigabyte Int
+  deriving stock (Eq, Show)
+  deriving newtype (Read, Aeson.FromJSON)
+
+toBytes :: Gigabyte -> Int
```

Comment:
I wonder if we should use a `newtype` for `Bytes` so that we make clear what are we converting to.

Reply:
I wondered myself too, but I have chosen not to -- we only use the `toBytes` function once where we require an `Int`. I have added a small comment to `toBytes` to clarify its use

```haskell
LMDBBackingStore $ maybe id (\y x -> x { lmdbMapSize = toBytes y }) newLimit defaultLMDBLimits
```